### DISCUSSION TITLE
Resizable : true but only one side

### DISCUSSION THREAD

**Discussion by jloguercio** [GrapesJS/grapesjs#4053] [Q&A] [answered]
Hi, There is a way to make a componen resisable but only one side? for example only bottom side? something like :

resisable : {
       bottom : true
}

Thanks

**Comment by Ju99ernaut** [comment, marked as answer]
You can use something like this:
```js
resizable: {
    tl: 0,
    tc: 0,
    tr: 0,
    cl: 0,
    cr: 0,
    bl: 0,
    br: 0,
};
```

This leaves only the `bc(bottom-centre)` option as true

**Reply by jloguercio** [reply]
Thank you very much, works like a charm 👍
